v22.1/vault-db-secrets-tutorial.md

```diff
@@ -0,0 +1,472 @@
+
+- The CockroachDB CLI installed locally. [Install CockroachDB](install-cockroachdb-mac.html)
+- The Vault CLI installed locally. [Install Vault](https://www.vaultproject.io/downloads)
+- Access as [`admin` SQL user](security-reference/authorization.html#admin-role) to a CockroachDB cluster. You may either [Create a {{ site.data.products.serverless }} cluster](../cockroachcloud/create-a-serverless-cluster.html) or [Start a Local Cluster (secure)](../{{site.versions["stable"]}}/start-a-local-cluster.html), however, for simplicity, this tutorial will demonstrate the procedure using {{ site.data.products.db }} and assume the user is doing the same. In either case you must have the public CA certificate for your cluster, and a username/password combination for the root SQL user (or another SQL user with the admin role).
+  You can create a {{ site.data.products.db }} cluster through the [{{ site.data.products.db }} console](https://cockroachlabs.cloud/).
+- Access to a Vault cluster with an admin token. You may either spin up a [free cluster in HashiCorp cloud](https://learn.hashicorp.com/collections/vault/cloud) or [start a development cluster locally](https://learn.hashicorp.com/tutorials/vault/getting-started-dev-server).
+  You can create a Vault cluster through the [HashiCorp Vault Cloud console](https://portal.cloud.HashiCorp.com/services/vault).
+
+
+## Introduction
+
+Sound strategy and proper tooling for managing database credentials are essential to the overall strength of your security posture. Managing your CockroachDB database credentials with HashiCorp Vault affords several security advantages:
+
+- General advantages of delegated, consolidated secrets management, as opposed to the industry standard of the past, which involves storing credentials in a variety of shared and individually owned file-systems, repositories and data stores, and sharing them via email, messaging, and other means of network communication:
+	- Access is managed by a single coherent set of policies.
+	- Access events leave an audit trail.
+	- Secrets are properly encrypted in flight and in transit.
+
+- Advantages of Vault's [Dynamic Secrets](https://www.vaultproject.io/use-cases/dynamic-secrets):
+	- Database credentials are generated and issued only on demand, from pre-configured templates, for specific clients and short validity durations, further minimizing both probability of credential compromise and possible impact of any compromise that does occur.
+	- Diligent rotation and revocation practices are automated by implication, requiring no additional effort or oversight.
+
+## Preparing your shell for CockroachDB and Vault admin tasks
+
+In this phase of the tutorial we will act as an administrator for our organization, provisioning access for a class of database clients.
+
+You'll need admin credentials for both a Vault cluster and a CockroachDB cluster, allowing you to create and manage access for roles in both domains.
```

Comment:
General **global** comment: Do you want to keep the second-person pronoun singular or plural throughout. Right now it's a mix, which I think could be fine, but maybe with more definition. For example, "we" in the step intros, and then "you"/imperative in the directions/instructions proper.